src/pages/Filter/Filter.scss

```
@@ -0,0 +1,302 @@
+$color-main: #fb7d27;
+$color-white: #ececec;
+$color-grey: #9b9b9b;
+
+div.UpperBox {
```

```suggestion
.UpperBox {
```
- 태그 셀렉터가 없더라도 동일한 결과일 것 같습니다

---

네 수정했습니다 .